### PR TITLE
Update Dyno -> AWS

### PR DESCRIPTION
**Pull request by tcql**

### CODE REVIEW

**Comment by tcql** [comment]
bumps dyno to 1.3, which is using aws-sdk 2.7. 

Dyno 1+ passes through most functions straight to aws-sdk's DocumentClient, so the format of all of the db queries changed. I'd appreciate some 👀  to make sure I translated everything okay. It seems to work 🆗  in my testing

cc @mick

File: lib/db.js [hunk 1]
```diff
@@ -49,69 +86,152 @@ module.exports = function(dyno, kinesis, config) {
...
+      ReturnValues: 'ALL_NEW'
+    }, function (err, response) {
+      var updated = err ? response : response.Attributes;
+      return cb(err, updated);
+    });
   };
 
   db.cleanupInstances = function(cb) {
-    dyno.query({type:{EQ:'instance'}}, {pages:0}, function(err, instances){
+    dyno.query({
```
Comment (mick) [inline lib/db.js]: wont be a problem at any scale we have run kine so far, if there are enough instances we'd want to have paging here

Comment (mick) [inline lib/db.js]: I think the new dyno now supports paging

**Comment by mick** [comment]
cool thanks for upgrading this. we should add paging back, but otherwise lgtm

**Pull request merged**